Source/Scene/Expression.js

```diff
@@ -31,8 +31,10 @@ define([
         return this._runtimeAst.evaluate(feature);
     };
 
-    function Node(value) {
+    function Node(value, left, right) {
```

Comment:
I think `left`/`right` is fine for now, but it will not scale later to expressions like `[1, 2, 3]` if we decide to support them.